templates/react/components/foo/List.js

```diff
@@ -37,7 +38,10 @@ class List extends Component {
             <tr>
               <th>Id</th>
 {{#each fields}}
+  {{#contains name "id" }}
```

Comment:
that's right, I tried to get a way to have @id first in the List & Show components. 
Do you know how to get it ?